src/gamma_inc.jl

```diff
@@ -0,0 +1,580 @@
+# Volume 12 Issue 4, Dec. 1986 Pages 377-393
+# doi>10.1145/22721.23109
+
+"""
+    gamma_p(a,x,IND)
+    
+    DLMF: https://dlmf.nist.gov/8.2#E4 , https://dlmf.nist.gov/8.2#E5
+    Wiki: https://en.wikipedia.org/wiki/Incomplete_gamma_function
+    IND --> Accuracy desired ; IND=0 means 14 significant digits accuracy , IND=1 means 6 significant digit and IND=2 means only 3 digit accuracy suffices.
+    gamma_p(a,x) or P(a,x) is the Incomplete gamma function ratio given by : ``1/\\Gamma (a) \\int_{0}^{x} e^{-t}t^{a-1} dt``
+"""
+function gamma_p(a::Float64,x::Float64,ind::Integer)
+    iop = ind + 1
+    acc = acc0[iop]
+    if a<0.0 || x<0.0
+        throw(DomainError((a,x,ind,"`a` and `x` must be greater than 0 ---- Domain : (0,inf)")))
+    elseif a==0.0 && x==0.0
+        throw(DomainError((a,x,ind,"`a` and `x` must be greater than 0 ---- Domain : (0,inf)")))
+    elseif a*x==0.0 
+        if x<=a
+            return 0.0
+        else
+            return 1.0
+        end               
+    end
+    
+    if a >= 1.0
+        @goto l10
+    elseif a == 0.5
+        @goto l320
+    elseif x < 1.1
+        @goto l110     
+    end
+    r = rgammax(a,x)
+    if r == 0.0
+        return 1.0
+    else
+        @goto l170    
+    end
+
+    @label l10
+     if a >= big1[iop]
+        @goto l20
+     elseif a > x || x >= x0[iop]
+        @goto l30 
+     else
+        twoa = a + a
+        m = trunc(Int,twoa)
+        if twoa != Float64(m)
+            @goto l30 
+        end
+        i = a
+        i = trunc(Int,i)
+        if a == Float64(i)
+            @goto l140
+        else
+            @goto l150
+        end            
+
+     end
+
+    @label l20
+     l = x/a
+     if l == 0.0
+        return 0.0
+     end
+     s = 1.0 - l
+     z = -logmxp1(l)
+     if z >= 700.0/a
+        return 0.0
+     end
+     y = a*z
+     rta = sqrt(a)
+     if abs(s) <= e0[iop]/rta
+        @goto l250
+     end
+
+     if abs(s) <= 0.4
+        @goto l200
+     end
+    
+    @label l30
+      r = rgammax(a,x)
+      if r == 0.0
+        if x <= a
+            return 0.0
+        else
+            return 1.0
+        end
+      end 
+      if x <= max(a,alog10)
+        @goto l50
+      elseif x < x0[iop]
+        @goto l170
+      else
+        @goto l80
+      end
+
+     #----TAYLOR SERIES FOR P/R---- 
+    @label l50
+     apn = a + 1.0
+     t = x/apn
+     wk[1] = t
+     loop=2
+     while true
+        if loop > 20
+            break
+        end
+        
+        apn = apn + 1.0
+        t = t*(x/apn)
+        if t <= 1.0e-3
+            @goto l60
+        end
+        wk[loop] = t
+        loop = loop + 1
+     end
+    loop=20
+    @label l60
+     sum = t
+     tol = 0.5*acc #tolerance
+     while true
+        apn = apn+1.0
+        t = t*(x/apn)
+        sum = sum + t
+        if t <= tol
+            break
+        end
+     end
+     maxd = loop - 1
+     for j = 1:maxd
+        loop = loop - 1
+        sum = sum + wk[loop]
```

Comment:
change the `for` loop iterator to matching the index, i.e.
```suggestion
    for k = loop-1:1
        sum += wk[j]
```
(also rename `sum`)